app/api/server_version.py

```diff
@@ -1,13 +1,15 @@
 from flask import jsonify, Blueprint
+from datetime import datetime
 
+SERVER_VERSION = '2.2.3-SNAPSHOT'
```

Comment:
It's not 2.2.3-SNAPSHOT

That was just an example

Reply:
Let me know what is the version I should put.

Reply:
See README